variables.tf

```
@@ -49,6 +49,12 @@ variable "policy_documents" {
   default     = []
 }
 
+variable "policy_document_count" {
+  type        = "string"
+  description = "Number of policy documents (length of policy_documents list)."
+  default     = "1"
```

```suggestion
  default     = 0
```

Make default 0 since we have a default empty list, and this https://github.com/cloudposse/terraform-aws-iam-role/pull/9/files#diff-7a370d8342e7203b805911c92454f0f4R52 will create the resources even if no policy documents are provided